docs/development/affiliated-packages.rst

```diff
@@ -491,17 +490,13 @@ files manually`_ section since this explains what many of the files do.
       for mod_name in MOCK_MODULES:
           sys.modules[mod_name] = Mock()
 
-#. You're now ready to start doing actual work on your affiliated package.  You
+#. You're now ready to start doing actual work on your package.  You
    will probably want to read over the developer guidelines of the Astropy
    documentation, and if you are hosting your code in GitHub, you might also
    want to read the `Github help <https://help.github.com/>`_ to ensure you know
    how to push your code to GitHub and some recommended workflows that work for
    the core Astropy Project.
 
-#. Once you have started work on the affiliated package, you should register
```

Comment:
Is there a different place where this information could be found?

Reply:
yes, at the affiliated package page of astropy.org: http://www.astropy.org/affiliated/

Reply:
I would even suggest to change the name of this page from ``affiliated-packages`` to ``package-template``, but that would potentially break links if anyone has them hard wired.

Reply:
Yeah, unless you want to retain the old page and just have old page link to new page... That is up to you. Otherwise, I think this is good to merge. Thanks!